guessTheNumber.js

```diff
@@ -0,0 +1,19 @@
+var playerNumber = prompt("I'm thinking of a number between 1 and 100. Can you guess the number I'm thinking of?")
```

Comment:
Consider using ES6 `let` or `const` - `const` is more appropriate here since this number should not change.